arcanist/arcanist/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: 
+
+pkgname=('arcanist')
+pkgdesc="Command line interface for Phabricator"
+arch=('i686' 'x86_64')
+url="https://secure.phabricator.com/book/phabricator/article/arcanist/"
```

Comment:
The `url` is intended to make `source` more parameterized, not to provide URLs for further reading.